views/book.js

```diff
@@ -0,0 +1,54 @@
+//var used for global scope availability
+var app = app || {};
+// noinspection JSVoidFunctionReturnValueUsed
+app.BookItem = Backbone.View.extend({
+	//<tr> because I am inserting element in a table
+	tagName: 'tr',
+	//when one of the input boxes corresponding to this model is changed update model
+	events : {
+		"change input" : 'updateModel',
+	},
+	/**
+	 * Renders the html for a book
+	 * @returns {app.BookItem}
+	 */
+	render: function () {
```

Comment:
Suggestion: have the HTML load from a template file.

Reply:
how do I send the data then? or I don't and just edit the elements with jQuery and add data that way?

Reply:
good suggestion Ovidiu, we'll try to do that a bit later.

Reply:
done :)